packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -640,6 +650,15 @@ abstract class FlutterCommand extends Command<void> {
       }
     }
 
+    String analyzeSize;
+    if (argParser.options.containsKey(FlutterOptions.kAnalyzeSize)
+      && boolArg(FlutterOptions.kAnalyzeSize)
+      && !globals.platform.isWindows) {
+      final File file = globals.fsUtils.getUniqueFile(globals.fs.currentDirectory, 'flutter_size', '.json');
```

Comment:
'.json' -> 'json'. Right now the output file name looks like `flutter_size_01..json`

Reply:
Done